Tasks/GradleV2/gradletask.ts

```diff
@@ -47,7 +47,7 @@ function publishTestResults(publishJUnitResults: boolean, testResultsFiles: stri
         }
 
         if (!matchingTestResultsFiles || matchingTestResultsFiles.length === 0) {
-            tl.warning('No test result files matching ' + testResultsFiles + ' were found, so publishing JUnit test results is being skipped.');
+            console.log('No test result files matching ' + testResultsFiles + ' were found, so publishing JUnit test results is being skipped.');
```

Comment:
I think this should be (ie. should have been) localized if it prints to all users.  Only `debug` messages don't need localization, no?

Reply:
Good point, the previous warning was also not localized which is incorrect.